blogs/dns_tracking/dns_tracking.md

```diff
@@ -0,0 +1,136 @@
+# Network Observability Per Flow DNS tracking
+
+![logo](./images/dns_tracking_logo.png)
+
+By: Julien Pinsonneau, Mehul Modi and Mohamed S. Mahmoud
+<style>body {text-align: justify}</style>
+
+In today's interconnected digital landscape, Domain Name System (DNS) tracking
+plays a crucial role in networking and security.
+DNS resolution is a fundamental process that translates human-readable domain
+names into IP addresses, enabling communication between devices and servers.
+However, this process also presents opportunities for monitoring and analysis,
+which can be achieved through innovative technologies like
+eBPF (extended Berkeley Packet Filter).
+In this blog post, we'll delve into the world of DNS tracking using eBPF
+tracepoint hooks, exploring how this powerful combination can be used for
+various purposes, including network monitoring and security enhancement.
+
+## Understanding DNS Resolution
+
+Before diving into the specifics of eBPF tracepoint hooks, let's briefly
+recap how DNS resolution works.
+When you enter a website's domain name (e.g., www.example.com) in your
```

Comment:
Can you figure out a way to remove the clickable link?  Maybe:
```suggestion
When you enter a website's domain name, such as www.example.com, in your
```

Reply:
https://github.com/netobserv/documents/pull/53/commits/267b3b733cb84342bd042862820bcfcd03c40988